model-dmdocument/src/main/java/gov/nasa/pds/model/plugin/DMDocument.java

```diff
@@ -456,9 +454,22 @@ public static void main(String args[]) throws Throwable {
     processArgumentParserNamespacePhase1(argparse4jNamespace);
 
     // first get the environment variables
-    getEnvMap();
-    dataDirPath = lPARENT_DIR + "/Data/";
-
+    String USER_DIR = System.getProperty("user.dir");

```

Comment:
@jshughes unfortunately, this doesn't work the way we want it to. By using the `user.dir` property, you are assuming the user is running the software from a specific location. However, if they are running it from their path or from some other directory, it won't work.

we probably want to do something like what [validate does here](https://github.com/NASA-PDS/validate/blob/main/src/main/resources/bin/validate#L66) where we set a new java property (`-Dresources.home=${PARENT_DIR}/resources`) and then we get that property in the launcher code (`System.getProperty("resources.home")`). So in this case you can add something like `-Ddata.home=${PARENT_DIR}/Data` to the LDDTool command-line script, and then pull that property in your code.

You can maybe use the `user.dir` as a fallback in the event `data.home` is not set.

hopefully that makes sense.